### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hopkiw**
Update `README.md` to reflect organizational changes and currently supported distribution details

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -279,16 +279,17 @@ provided Python versions.
 
 Distro       | Package Type | Python Version | Init System
 ------------ | ------------ | -------------- | -----------
-Debian 8     | deb          | 2.7            | systemd
-Debian 9     | deb          | 3.5 or 2.7     | systemd
+SLES 12      | rpm          | 2.7            | systemd
+SLES 15      | rpm          | 2.7            | systemd
```
Comment (zmarano) [inline README.md]: 3.6 for SLES 15

File: README.md [hunk 2]
```diff
@@ -302,10 +303,13 @@ We build the following packages for the Linux guest environment.
...
+    *  The on-boot resize scripts for root partition.
 
-The package source for Debian and RPM specs for Enterprise Linux 6 and 7 are
-included in this project. There are also
-[Daisy](https://github.com/GoogleCloudPlatform/compute-image-tools/tree/master/daisy)
+The package sources (RPM spec files and Debian packaging directories) are also
+included in this project. There are also [Daisy](https://github.com/GoogleCloudPlatform/compute-image-tools/tree/master/daisy)
 workflows for spinning up GCE VM's to automatically build the packages for
 Debian, Red Hat, and CentOS. See the [README](packaging/README.md) in the
 packaging directory for more details.
```
Comment (zmarano) [inline README.md]: Update the Version Updates section below to include any other specifics for os login etc... just as a point of reference. I believe this current language is actually inaccurate? (or maybe I'm wrong).

File: README.md [hunk 3]
```diff
@@ -321,64 +325,35 @@ The method for making version updates differs by package.
 
 #### Package Distribution
 
-The deb and rpm packages used in some GCE images are published to Google Cloud
-repositories. Debian 8 and 9, CentOS 6 and 7, and RHEL 6 and 7 use these
-repositories to install and update the `google-compute-engine`, and
-`python-google-compute-engine` (and `python3-google-compute-engine` for Python 3)
-packages. If you are creating a custom image, you can also use these repositories
-in your image.
+The deb and rpm packages are published to Google Cloud repositories. Debian,
```
Comment (zmarano) [inline README.md]: Include google-compute-engine-oslogin here

**Comment by illfelder** [comment]
Please don't ignore comments on PRs.

**Pull request merged**